zero_bin/ops/src/lib.rs

```diff
@@ -25,6 +28,22 @@ use zero_bin_common::{debug_utils::save_inputs_to_disk, prover_state::p_state};
 
 registry!();
 
+#[cfg(feature = "test_only")]
+#[derive(Deserialize, Serialize, RemoteExecute)]
+pub struct BatchTestOnly {}
```

Comment:
is there any reason why you're not using `save_inputs_on_error` field in this? Even though we technically _can't_ log yet the inputs upon error because the iterator is panicking, we are aiming at changing this logic before the release, which would allow to save inputs even in testing mode.

Reply:
Yeah, I assumed we would add it when logging is available but we can have it there already. I also mimicked the error handling of the other operations for the future.